src/components/notifications/DingDing.vue

```diff
@@ -6,6 +6,12 @@
         <label for="secretKey" class="form-label">{{ $t("SecretKey") }}<span style="color: red;"><sup>*</sup></span></label>
         <input id="secretKey" v-model="$parent.notification.secretKey" type="text" class="form-control" required>
 
+        <label for="isAtAll" class="form-label">{{ $t("isAtAll") }}<span style="color: red;"><sup>*</sup></span></label>
+        <select id="isAtAll" v-model="$parent.notification.isAtAll" class="form-select" required>
+            <option value="false">{{ $t("false") }}</option>
+            <option value="true">{{ $t("true") }}</option>
+        </select>
```

Comment:
What I meant was something like this, but
- with the respective translations in `en.json`,
- the respective changes to `dingding.js` and
- testing that I did not make a typo in the github webeditor ^^^

What do you think about the label? Is this self-explaiantory enough? Do we need to add an helptext? (would not know what to put there, but still would like to have your oppinon on this one)


```suggestion
        <label for="metioning" class="form-label">{{ $t("Mentioning") }}<span style="color: red;"><sup>*</sup></span></label>
        <select id="metioning" v-model="$parent.notification.metioning" class="form-select" required>
            <option value="nobody">{{ $t("Don't mention people") }}</option>
            <i18n-t keypath="Notify everyone" tag="option" value="@everyone">
                <code>@everyone</code>
            </i18n-t>
        </select>
```